manifests/init.pp

```diff
@@ -231,7 +231,7 @@
       path    => "${confdir}/puppet.conf",
       section => 'main',
       setting => 'hiera_config',
-      value   => $hiera_yaml,
+      value   => '$confdir/hiera.yaml',
```

Comment:
But you could set `$hiera_yaml` to a different value and by hardcoding this, it would not take effect.